tests/utils/test_df_ops.py

```diff
@@ -1,11 +1,13 @@
+
 def test_dfConverter():
```

Comment:
that's my useless test here. don't mind removing

Reply:
we'll leave for now, maybe we can expand it later